roles/deploy/defaults/main.yml

```diff
@@ -85,7 +85,8 @@ project_post_build_commands:
 # These are meant primarily for service restarts such as php5-fpm or memcached.
 project_post_finalize_commands:
   - if wp core is-installed; then wp eval 'wp_clean_themes_cache(); switch_theme(get_stylesheet());'; fi
-  - sudo service php5-fpm reload
+  - if dpkg --list | grep -Fq 'php5-fpm'; then sudo service php5-fpm reload; fi
+  - if dpkg --list | grep -Fq 'hhvm'; then sudo service hhvm restart; fi
```

Comment:
You could also do the same with `initctl list | grep -Fq 'hhvm'` or `php5-fpm`, which will return the status of the service (and its state). I think its a little bit cleaner too.